glean-core/ffi/LICENSE

```diff
@@ -0,0 +1,373 @@
+Mozilla Public License Version 2.0
```

Comment:
How unfortunate that we need multiple copies of our license at different directory levels.

Reply:
oh well, at least this file doesn't change :)